incubator/cockroachdb/values.yaml

```diff
@@ -0,0 +1,17 @@
+# Default values for cockroachdb.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Name: "cockroachdb"
+Image: "cockroachdb/cockroach"
+ImageTag: "latest" # Keep at latest until 1.0, then switch to a specific version
```

Comment:
I would ask that we keep it in incubator till we version the container. Production boom - cell phone nuts - not fun. Reliving first k8s production outage. Now going to sleep.

Thanks for the chat looks great! We missing any changes?